DOCS-4036: Add more frame and geometry content, remove arm no code (#4411)
viamrobotics/docs: docs/operate/mobility/move-arm/configure-additional.md

You can also take into account passive objects attached to the arm such as a camera mount to avoid collisions.

## Configure a gripper's frame

If you have a gripper, you can configure it to move with the arm.

> Configure the gripper's frame to describe its position and orientation relative to the arm:
> 
> 1. See [Configure a gripper](/operate/reference/components/gripper/) for instructions on configuring the gripper itself.

Reviewer: Just because of the change in the headline

```suggestion
If you haven't already configured your gripper, see [Configure a gripper].

Configure the gripper's frame to describe its position and orientation relative to the arm:
```

1. In the **CONFIGURE** tab, find the gripper's configuration card.

1. Click **+ Add frame**.

1. Select the arm's frame as the parent frame.

1. Enter the gripper frame's position and orientation relative to the center of the end of the arm.
   It is up to you to decide what part of the gripper you want to use as the gripper frame.

>    It can be convenient to use a point near the center of the gripper jaws as the gripper frame.

Reviewer: This sounds more like guidance. I was reading this wondering how I'd know if that was convenient or not in a specific case
```suggestion
   It is generally convenient to use a point near the center of the gripper jaws as the gripper frame.
```

>    When you make a call to the motion service to move the gripper to a location, the point you specify in this step is the part of the gripper that will move to that location.

Author: I am leaning towards leaving as-is to keep the sentence from getting as long and so that if someone has a different point they want to move, it's clear how to do that.

   }
   ```

1. Check whether your gripper has a kinematics file.

   Some grippers have [kinematics files](/operate/reference/kinematic-chain-config/) that describe the position and orientation of the jaws of the gripper as they move, relative to the gripper's base.

>    This file typically also contains geometries to represent the gripper's volume and avoid collisions between the gripper and its environment.

Author: ```suggestion
   This file typically also contains geometries that represent the gripper's volume to avoid collisions between the gripper and its environment.
```

   You can check whether your gripper has a kinematics file by looking in its module source code, or by using the [`GetKinematics` method](/dev/reference/apis/components/gripper/).

   If you have a kinematics file, you do not need to add any geometries to the gripper's frame, assuming the gripper appears as expected in the **VISUALIZE** tab.

   _If you do not have a kinematics file, see [Configure the geometry of the object](#configure-the-geometry-of-the-object) to configure a geometry for the gripper to avoid collisions between the gripper and its environment._

## Configure a camera

If you have a camera that can see the environment, configure the camera's frame to describe its position and orientation relative to the arm so that what the camera sees can be used to plan the arm's motion:

1. Configure the camera itself.
   For example, you can use an Intel Realsense camera by configuring the `realsense` model of the camera component according to its [documentation](https://app.viam.com/module/viam/realsense).

1. In the **CONFIGURE** tab, find the camera's configuration card.

1. Click **+ Add frame**.

> 1. Edit the frame depending on where the camera is mounted:

Author: I don't think that's in the docs. Should it be? I suppose last I heard (a little bit ago) was that it was still essentially beta

> 1. Edit the frame depending on where the camera is mounted:

Reviewer 2: Just checked with John on the current state - it sounds like it works well enough and the README is filled out (https://github.com/viam-modules/frame-calibration). The main issue is that the PoseTracker component type (which is one of the component types it depends on) isn't in the docs. Not sure if it's worth opening that can of worms right now. Internally we are pretty good about sharing knowledge of its existence, so it'd mainly be for external folks.

> 1. Edit the frame depending on where the camera is mounted:

Author: Okay I'll ask around, and I just made a docs ticket
